time-y1-nozzle.sh

```diff
@@ -3,79 +3,133 @@
 set -e
 set -x
 
+TIMING_HOME=$(pwd)
+TIMING_HOST=$(hostname)
+TIMING_DATE=$(date)
+
+# -- installs conda env, dependencies and MIRGE-Com via *emirge*
 git clone https://github.com/illinois-ceesd/emirge.git
 cd emirge
-./install.sh
+./install.sh --env-name=nozzle.timing.env
```

Comment:
```suggestion
./install.sh
```
I think this should try to use "current conda".

Reply:
It is a problem if it wipes out the user's default, usual, environment.  It needs to make its own for this.  Any harm in making a timing-specific environment?

Reply:
Ah! Nvm. I misread this as "use this environment file".